Package.swift

```diff
@@ -245,7 +245,8 @@ let package = Package(
                 "BlockRuntime",
                 "CMakeLists.txt"
             ],
-            cSettings: coreFoundationBuildSettings
+            cSettings: coreFoundationBuildSettings,
+            linkerSettings: [.linkedLibrary("log", .when(platforms: [.android]))]
```

Comment:
This [was in the CMake config for Swift 5](https://github.com/swiftlang/swift-corelibs-foundation/blob/8d1a6794206637fee67f4a955853b0d1771746e2/CoreFoundation/CMakeLists.txt#L405), but was not brought over in the Foundation rewrite in Swift 6.

I still haven't needed it for the new CMake config, maybe because [it was set up as a framework in Swift 5](https://github.com/swiftlang/swift-corelibs-foundation/blob/8d1a6794206637fee67f4a955853b0d1771746e2/CoreFoundation/CMakeLists.txt#L130) and now [it's just a CMake static library](https://github.com/swiftlang/swift-corelibs-foundation/blob/9c03e2eded83ce7a1e1e24717af96b4c21e4d4d8/Sources/CoreFoundation/CMakeLists.txt#L15)?